flood_adapt/dbs_controller.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, database_path: Union[str, os.PathLike], site_name: str) -> No
         # self.update()
 
     # General methods
-    def get_aggregation_areas(self) -> list[GeoDataFrame]:
+    def get_aggregation_areas(self) -> dict:
```

Comment:
Wouldn't it be useful to also have a function where you can get a single area instead of doing all transitions on all area's